### ISSUE TITLE
Error: undefined for type : " ", type may not be implemented in usePersistField.

### ISSUE THREAD

**Issue by harshit078** [twentyhq/twenty#8529]
## Bug Description

- When interacting with fields with RightDrawer action (Opportunities,People,Companies), an error is thrown and logged onto Sentry - 

`Uncaught Error: Invalid value to persist: undefined for type : "  ", type may not be implemented in usePersistField.`

## Current Behaviour

https://github.com/user-attachments/assets/9929dae7-dddc-4581-ae43-fc3fd5d5a841

![Image](https://github.com/user-attachments/assets/6373bb58-bbe2-484a-a902-69c5710e2fef)

**Comment by FelixMalfait** [comment]
Good catch thanks! We probably introduced it here: https://github.com/twentyhq/twenty/pull/8474/files#diff-7121b15abfc45cafa494588627867973bc846956bb3c586d8f6fb223c87b6bd1R57

**Comment by FelixMalfait** [comment]
@nicolasrouanne I was wrong (again! sorry), I think this was introduced here https://github.com/twentyhq/twenty/pull/7729 cc @lucasbordeau fyi

**Comment by lucasbordeau** [comment]
The fix should be easy to do in usePersistField, I'll try to refactor the logic soon https://github.com/twentyhq/twenty/issues/8361 But if it's urgent, the fix can be done by using isDefined at the right place.

**Comment by charlesBochet** [comment]
I cannot reproduce anymore, closing this one
